src/components/graph/menus/model-node-editor.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React, { useState } from 'react';
+import { NativeSelect, TextField } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+import { useIntl } from 'react-intl';
+
+const ModelNodeEditor = ({ selectedNode }) => {
+    const intl = useIntl();
+
+    const [nameValue, setNameValue] = useState(selectedNode.data?.label);
+    const [modelType, setModelType] = useState(selectedNode.modelType);
+
+    const updateNameValue = (event) => {
+        setNameValue(event.target.value);
+    };
+
+    const updateModelType = (event) => {
+        setModelType(event.target.value);
+    };
+
+    const handleValidate = (event) => {
+        selectedNode.data.label = nameValue;
+        selectedNode.modelType = modelType;
+    };
+
+    return (
+        <>
+            <Grid container spacing={3} alignItems="flex-end">
+                <Grid item xs={12}>
+                    <TextField
+                        label={intl.formatMessage({ id: 'Name' })}
+                        value={nameValue}
+                        onChange={updateNameValue}
+                    />
+                </Grid>
+            </Grid>
+            <Grid container spacing={3} alignItems="flex-start">
+                <Grid item xs={12}>
+                    <NativeSelect value={modelType} onChange={updateModelType}>
+                        <option aria-label="None" value="" />
+                        <option value={'LOADFLOW'}>
+                            {intl.formatMessage({ id: 'LoadFlow' })}
+                        </option>
+                        <option value={'SECURITY_ANALYSIS'}>
+                            {intl.formatMessage({ id: 'SecurityAnalysis' })}
+                        </option>
+                    </NativeSelect>
+                </Grid>
+            </Grid>
+            <Grid container spacing={3} alignItems={'flex-start'}>
+                <Grid item>
+                    <Button onClick={handleValidate}>
```

Comment:
Do we need a cancel button here ?

Reply:
Node editor is cancelled when deselecting edited node. Anyway, the node editor will be replaced in a further version by another component.